### ISSUE TITLE
Duplicate links

### ISSUE THREAD

**Issue by Xiaobonor** [miantiao-me/Sink#15]
Should duplicate links and the same settings be limited to one link, and all future links should be directed to the same short link?

![image](https://github.com/ccbikai/Sink/assets/114270373/d5fded48-f82e-412e-ada5-3470856eb758)

**Comment by teatall** [comment]
Yes, this is very important.

**Comment by tabp0le** [comment]
I disagree. Sometimes it's really useful to have duplicate links, with a different short link and their own analytics per link. An example is, if you are sharing a link with multiple people or in multiple places, and you want to track the traffic and clicks from each source.

**Comment by miantiao-me** [comment]
It's reasonable for a link to be able to create multiple short links; we won't consider this feature for now
